web/pom.xml

```diff
@@ -237,10 +239,10 @@
                 <version>1.7</version>
                 <executions>
                     <execution>
-                        <id>ui-package</id>
-                        <phase>prepare-package</phase>
+                        <id>ui-download-dependencies</id>
+                        <phase>clean</phase>
```

Comment:
Why on `clean` phase?

Reply:
A simple way not having to execute this everytime when building locally.